### ISSUE TITLE
Firing range targets, when shot too often, crash clients once they come in viewport range of it (but only once)

### ISSUE THREAD

**Issue by SonixApache** [d3athrow/vgstation13#20284]
#### Description of issue

The targets on the firing range crash people after a certain amount of shots has been fired at them.
I don't know the exact number, but two or three mags of glock should be enough.
I think it's the amount of overlays on the sprite.

#### Steps to reproduce
Go to firing range, empty a few guns into a target, leave area and return?
I'm not sure how to *cause* it, I've only ever felt and heard the effects.

Often when dedchat reports that they had crashed, I'll ask them if they happened to be near the brig when it happened, and more than often the answer is yes.

#### Specific information for locating
 `/obj/item/target`

#### Length of time in which bug has been known to occur
Month or two?

#### Server revision
0732f797e3e5915fcf820e1765b84c813351a755

**Comment by Exxion** [comment]
Note that just because it's a BYOND issue doesn't mean we can't fix it
We can combine the overlays into one with icon procs

**Comment by SGuivenne** [comment]
Duplicate of #19821 

edit: Thanks damian senpai.

**Comment by SGuivenne** [comment]
Thanks.

**Comment by SonixApache** [comment]
i tested this recently and it is no longer an issue (sadly)
